grr/server/grr_response_server/gui/ui/tsconfig.json

```diff
@@ -29,7 +29,9 @@
     ],
     "paths": {
       "@app/*": ["./*"]
-    }
+    },
+    "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
Why do we need to turn on the esModuleInterop setting?

Reply:
This was required for the json import (not sure what exactly), but there's no need for it now.